### ISSUE TITLE
Designer switch dialog "switch" button should use standard highlight color

### ISSUE THREAD

**Issue by bjkeller** [klavinslab/aquarium#364]
In the designer tab, when you switch between plans a dialog opens. The "switch" button should be the dark blue highlight color, but is grey.

**Comment by lekhacminhphuong** [comment]
``` js
var dialog = $mdDialog.confirm()
              .title('Switch Plan?')
              .textContent(msg ? msg : "The current plan will be saved")
              .ariaLabel('Save First?')
              .ok('Switch')
              .cancel('Cancel')
              .clickOutsideToClose(true); 

 $(".md-primary.md-confirm-button.md-button.md-ink-ripple.md-default-theme").attr('id', 'switch_dialog')
```
I use Jquery to access to the "Switch" button, adding "switch_dialog" id attribute for the button, and using this id to style the button in browser.scss.

However, I did not get the color change on the "Switch" button.

**Comment by lekhacminhphuong** [comment]
Fixed by d8658aa

**Comment by lekhacminhphuong** [comment]
Fixed by 4898ab8
